app/themes/3column/ui/app/root/root.html

```diff
@@ -1,68 +1,25 @@
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-12">
-        <div class="social-links">
-          <a href="https://www.facebook.com/marklogic" title="Facebook" rel="external">
-            <img src="images/facebook.svg" alt="Facebook" title="Facebook"></a>
-          <a href="https://twitter.com/marklogic" title="Twitter" rel="external">
-            <img src="images/twitter.svg" alt="Twitter" title="Twitter"></a>
-          <a href="http://www.linkedin.com/company/marklogic" title="LinkedIn" rel="external">
-            <img src="images/linkedin.svg" alt="LinkedIn" title="LinkedIn"></a>
-          <a href="https://www.flickr.com/photos/marklogicuc/" title="Flickr" rel="external">
-            <img src="images/flickr.svg" alt="Flickr" title="Flickr"></a>
-          <a href="https://www.youtube.com/channel/UCT2Q5qhC2gmd89K9qpV2TrQ" title="YouTube" rel="external">
-            <img src="images/youtube.svg" alt="YouTube" title="YouTube"></a>
-          <a href="https://plus.google.com/+MarklogicCo/posts" title="Google+" rel="external">
-            <img src="images/google.svg" alt="Google+" title="Google+"></a>
-        </div>
-      </div>
+<div>
+  <nav class="navbar navbar-default navbar-fixed-top" role="navigation">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <ml-user></ml-user>
+      <a class="navbar-brand" href="/"></a>
     </div>
-    <div class="row">
-      <div class="col-sm-12">
-        <ul id="menu-footer-navigation-logged-in" class="nav">
-          <li id="menu-item-140" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-140">
-            <a target="_blank" href="http://www.marklogic.com">MarkLogic.com</a>
-          </li>
-          <li id="menu-item-71" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-71">
-            <a target="_blank" href="http://developer.marklogic.com">MarkLogic Developers</a>
-          </li>
-        </ul>
+    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+      <div class="navbar-form navbar-right" ng-show="user.authenticated">
+        <ml-remote-input></ml-remote-input>
       </div>
-    </div>
-    <div class="row copyright">
-      <div class="col-sm-12 text-muted">
-        Copyright <sup>&copy;</sup> {{ ctrl.currentYear }} MarkLogic Corporation. MARKLOGIC <sup>&reg;</sup> is a registered trademark of MarkLogic Corporation.
-      </div>
-    </div>
+    </div><!-- /.navbar-collapse -->
+  </nav>
+  <!-- Add your site or application content here -->
+  <div class="container-fluid" ui-view></div>
+
+  <div id="footer">
+    <div class="text-right"><small>Powered by MarkLogic</small></div>
```

Comment:
You are loosing a lot of the footer here. A bit too much I'd say. Also note the ctrl.currentYear insertion for the copyright notice..